sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/Azure.ResourceManager.SecurityInsights.csproj

```diff
@@ -7,9 +7,4 @@
     <Description>Microsoft Azure Resource Manager client SDK for Azure resource provider Microsoft.SecurityInsights.</Description>
     <PackageTags>azure;management;arm;resource manager;securityinsights</PackageTags>
   </PropertyGroup>
-
-  <ItemGroup>
-    <!--This is needed until we solve this issue https://github.com/Azure/azure-sdk-for-net/issues/32395-->
-    <PackageReference Include="Microsoft.Azure.AutoRest.CSharp" VersionOverride="3.0.0-beta.20221102.1" PrivateAssets="All" />
-  </ItemGroup>
```

Comment:
Remove the version ping to avoid build faulire in CI when an old version generator uses the latest shared Azure.Core code. We need to evaluate whether the mentioned issue was resolved to really remove the version ping.